amundsen_application/static/js/ducks/dashboard/sagas.ts

```diff
@@ -8,10 +8,11 @@ import { GetDashboard, GetDashboardPreview, GetDashboardPreviewRequest } from '.
 export function* getDashboardWorker(action): SagaIterator {
   try {
     const uri = action.payload.uri;
-    const dashboard = yield call(API.getDashboard, uri);
-    yield put(getDashboardSuccess(dashboard));
-  } catch (e) {
-    yield put(getDashboardFailure());
+    const { data, status } = yield call(API.getDashboard, uri);
+    yield put(getDashboardSuccess(data.dashboard, status));
+  } catch (error) {
+    const { status } = error;
```

Comment:
Is the status not on `error.response.status`?

Reply:
Example: https://github.com/lyft/amundsenfrontendlibrary/blob/master/amundsen_application/static/js/ducks/tableMetadata/api/v0.ts#L141-L159

I know we had at least a couple bugs in the past from not processing the `AxiosError` correctly. Type info: https://github.com/axios/axios/blob/master/index.d.ts#L85

Reply:
I initially followed the response in `getPreviewDataWorker` which did 
```
catch (error) {
    const { data, status } = error;
```
But I did not see the code mentioned in your example which returned a cleaned error object.